wear/src/main/kotlin/au/com/shiftyjelly/pocketcasts/wear/di/WearNetworkModule.kt

```diff
@@ -33,6 +49,65 @@ object NetworkModule {
         coroutineScope
     )
 
+    @Provides
+    @Singleton
+    fun provideNetworkingRules(): NetworkingRules = PocketCastsNetworkingRules
+
+    @Provides
+    @Singleton
+    fun provideNetworkingRulesEngine(
+        networkRepository: NetworkRepository,
+        networkingRules: NetworkingRules,
+    ): NetworkingRulesEngine =
+        NetworkingRulesEngine(
+            networkRepository = networkRepository,
+            logger = NetworkStatusLogger.Logging,
+            networkingRules = networkingRules,
+        )
+
+    @Provides
+    @Singleton
+    fun provideHighBandwidthNetworkMediator(
+        connectivityManager: ConnectivityManager,
+        @ForApplicationScope coroutineScope: CoroutineScope,
+    ): HighBandwidthNetworkMediator =
+        StandardHighBandwidthNetworkMediator(
+            logger = NetworkStatusLogger.Logging,
+            networkRequester = NetworkRequesterImpl(connectivityManager),
+            coroutineScope = coroutineScope,
+            delayToRelease = 3.seconds,
+        )
+
+    @Provides
+    @Singleton
+    @DownloadCallFactory
+    fun provideDownloadWearCallFactory(
+        highBandwidthNetworkMediator: HighBandwidthNetworkMediator,
+        networkRepository: NetworkRepository,
+        networkingRulesEngine: NetworkingRulesEngine,
+        @DownloadOkHttpClient phoneCallFactory: OkHttpClient,
+        @ForApplicationScope coroutineScope: CoroutineScope,
+    ): Call.Factory {
+
+        return NetworkSelectingCallFactory(
+            networkingRulesEngine = networkingRulesEngine,
+            highBandwidthNetworkMediator = highBandwidthNetworkMediator,
+            networkRepository = networkRepository,
+            dataRequestRepository = null,
+            rootClient = phoneCallFactory,
+            coroutineScope = coroutineScope,
+            timeout = 5.seconds,
+        )
+    }
+
+    @Provides
+    @DownloadRequestBuilder
+    fun downloadRequestBuilder(): Request.Builder =
+        Request.Builder()
+            .requestType(RequestType.MediaRequest.DownloadRequest)
```

Comment:
This `RequestType` comes from the horologist library and setting it on the `Request.Builder` for wear requests is the only reason I need to inject it separately for the wear vs phone/automotive apps.